shared/oae/api/oae.api.util.js

```diff
@@ -1656,12 +1656,15 @@ define(['exports', 'require', 'jquery', 'underscore', 'oae.api.config', 'markdow
                 return '';
             } else {
 
+                input = input.replace(/\*/g, "\\*");
```

Comment:
We also have a number of comments in production that already escape `*` to avoid the italic behavior. Can we make sure that `*`s that are already escaped are not further escaped?